consensus/reactor.go

```diff
@@ -771,7 +771,7 @@ OUTER_LOOP:
 					peer.TrySend(StateChannel, cdc.MustMarshalBinaryBare(&VoteSetMaj23Message{
 						Height:  prs.Height,
 						Round:   prs.Round,
-						Type:    types.PrecommitType,
+						Type:    types.SIGNED_MSG_TYPE_PREVOTE_TYPE,
```

Comment:
```suggestion
						Type:    types.SIGNED_MSG_TYPE_PRECOMMIT_TYPE,
```

Reply:
a bit concerning tests still passed? will make the fixes.

Reply:
> a bit concerning tests still passed

want to write a test case?